### ISSUE TITLE
Can only edit one of the inputs...

### ISSUE THREAD

**Issue by fullymiddleaged** [ZhangGe6/onnx-modifier#129]
I seem to have an issue here where I can only edit one of my 2 fields: 'input_ids' and 'attention_mask'. The exe version will only let me edit the 'input_ids' dim sizes, not the 'attention_mask'. The drop down options also only say 'input_ids' - it's like its hard coded?

Any advice appreciated!

**Comment by fullymiddleaged** [comment]
This can be closed, I was being an idiot lol. I manually found the other input on the visual and amended it ok.
